synapse/storage/databases/main/event_push_actions.py

```diff
@@ -509,14 +441,18 @@ def _get_if_maybe_push_in_range_for_user_txn(txn):
         )
 
     async def add_push_actions_to_staging(
-        self, event_id: str, user_id_actions: Dict[str, List[Union[dict, str]]]
+        self,
+        event_id: str,
+        user_id_actions: Dict[str, List[Union[dict, str]]],
+        count_as_unread=False,
```

Comment:
not sure we need a default value here?

Reply:
we *do* need a type annotation though.

Reply:
> not sure we need a default value here?

That was so I didn't have to go and change the few tests that call this function directly but I guess I can also do that.

> we do need a type annotation though.

Ah, yeah, I forgot about that.